### DISCUSSION TITLE
Notes on doing eeg-expy dev with github codespaces

### DISCUSSION THREAD

**Discussion by JohnGriffiths** [NeuroTechX/EEG-ExPy#256] [General]
Github codespaces is fairly new (to me) tool and is clearly very handy for getting on with dev work. 

Starting a thread here to keep a note of some of the more and less obvious to-dos and whatnot of doing this.

**Comment by JohnGriffiths** [comment]
## Opening EEG-ExPy Dev in Up Codespaces

...

**Comment by JohnGriffiths** [comment]
## General Github PR Review

This is standard practice not specific to codespaces but always handy to have the main strokes clearly described, especially for newcomers. 

Reference page: 

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally

**Comment by JohnGriffiths** [comment]
##  Testing the docs build and debugging errors 

...

**Comment by JohnGriffiths** [comment]
## Testing out some code in a jupyter notebook
..

**Comment by oreHGA** [comment]
The command that worked for installation without a wxPython error `--pip install -e . --find-links=https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-20.04/`
